Cargo.toml

```diff
@@ -2,8 +2,8 @@
 members = ["geozero", "geozero-core", "geozero-shp", "geozero-cli", "geozero-bench"]
 default-members = ["geozero", "geozero-core", "geozero-shp", "geozero-cli"]
 
-#[patch.crates-io]
+[patch.crates-io]
 # Patch geozero dependencies (dev only)
-#geozero = { path = "geozero" }
-#geozero-core = { path = "geozero-core" }
+geozero = { path = "geozero" }
+geozero-core = { path = "geozero-core" }
```

Comment:
Note that I uncommented this - I expect that when I'm using the workspace, dependencies will be pulled in from the workspace. 

Or is there some reason to leave this commented?